code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -391,11 +391,7 @@ var/list/ai_verbs_default = list(
 	if(confirm == "Yes")
 		call_shuttle_proc(src)
 
-	// hack to display shuttle timer
-	if(emergency_shuttle.online())
-		var/obj/machinery/computer/communications/C = locate() in SSmachinery.processing_machines
-		if(C)
-			C.post_status("shuttle")
+	post_status("shuttle")

```

Comment:
Shouldn't this be:
```dm
if (emergency_shuttle.online())
    post_status("shuttle")
```
?

Reply:
I missed this bit. Sorry. It should be like you suggested.